matter: pull deferred attribute persister fix (#13465)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

The issues in this section are related to the :ref:`ug_matter` protocol.

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0

> KRKNWK-18221: Deferred attribute persister leaks heap memory buffer.

Reviewer: ```suggestion
KRKNWK-18221: Memory leak in the deferred attribute persister
```

  The ``DeferredAttributePersister`` helper class is used to limit the flash wear for applications that include fast-changing, persistent cluster attributes.

>   This class leaks 1-byte heap memory buffer for each deferred attribute write.
>   This can eventually lead to running out of heap memory and the need for rebooting a device to restore its normal operations.

Reviewer: ```suggestion
  This class leaks a 1-byte heap memory buffer for each deferred attribute write, which can eventually lead to running out of heap memory until the device is rebooted.
```
